src/theme/components/template.ts

```diff
@@ -68,27 +69,35 @@ const TemplateContentTopBottom = defineStyleConfig({
 const TemplateContentPrimary = defineStyleConfig({
   baseStyle: defineStyle({
     gridColumn: { base: "1", md: "1 / span 2" },
+    overflow: { base: "unset", md: "hidden" },
```

Comment:
This is where the issue @bigfishdesign13 flagged comes from in https://github.com/NYPL/nypl-ds-test-app/pull/119#issuecomment-2494008501